integration/tests/cook/test_basic.py

```diff
@@ -700,6 +700,41 @@ def test_detailed_for_groups(self):
         for k in detail_keys:
             self.assertIn(k, group_info)
 
+    def test_group_kill(self):
+        # Create and submit jobs in group
+        slow_job_wait = 1200
+        group_spec = self.minimal_group()
+        job_fast = util.minimal_job(group=group_spec['uuid'])
+        job_slow = util.minimal_job(group=group_spec['uuid'],
+                                    command='sleep %d' % slow_job_wait)
+        data = {'jobs': [job_fast, job_slow], 'groups': [group_spec]}
+        resp = util.session.post('%s/rawscheduler' % self.cook_url, json=data)
```

Comment:
We have `util.submit_job` for a single job, maybe we should create `util.submit_jobs` instead of repeating this pattern in multiple places

Reply:
I added both `util.minimal_jobs` and `util.submit_jobs`. It actually didn't help with this specific case, but I was able to use `util.minimal_jobs` in my new test since the jobs being created were homogeneous.

Reply:
I was thinking `submit_jobs` could take in the `job_specs` as an argument, so that it would work even for heterogeneous jobs

Reply:
And maybe it could also take in the `group_specs` (optionally)